src/main/resources/en/garments_sizes.yml

```diff
@@ -0,0 +1,11 @@
+en:
+  faker:
+    garments:
+      sizes:  ["XS", "S", "M", "L", "XL", "XXL", "XXXL"]
+      xs: ["XS"]
```

Comment:
Same for the mapping here, why would you map xs to XS?

Reply:
I remove senseless mapping!